apiserver/pkg/model/converter.go

```diff
@@ -111,3 +116,24 @@ func FromKubeToAPIComputeTemplates(configMaps []*v1.ConfigMap) []*api.ComputeTem
 	}
 	return apiComputeTemplates
 }
+
+func FromCrdToApiJobs(jobs []*v1alpha1.RayJob) []*api.RayJob {
+	apiJobs := make([]*api.RayJob, 0)
+	for _, job := range jobs {
+		apiJobs = append(apiJobs, FromCrdToApiJob(job))
+	}
+	return apiJobs
+}
+
+func FromCrdToApiJob(job *v1alpha1.RayJob) *api.RayJob {
+	pbJob := &api.RayJob{
```

Comment:
I remember you proposed to add name and namespace? When you convert the object, should the information be added?

Reply:
fixed, add name, namespace in job proto